zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/authorization/AddPermissionAuthorizationMultiPartitionTest.java

```diff
@@ -40,20 +40,22 @@ public class AddPermissionAuthorizationMultiPartitionTest {
   @Test
   public void shouldTestLifecycle() {
     // when
-    final var ownerKey =
+    final var userRecord =
         engine
             .user()
             .newUser("foo")
             .withEmail("foo@bar")
             .withName("Foo Bar")
             .withPassword("zabraboof")
-            .create()
-            .getKey();
+            .create();
+    final var ownerKey = userRecord.getValue().getUserKey();
+    final var ownerId = userRecord.getValue().getUsername();
 
     engine
         .authorization()
         .permission()
         .withOwnerKey(ownerKey)
+        .withOwnerId(ownerId)
```

Comment:
❓I think you missed to set the `ownerType` here, right ? I think in all test classes of this [commit](https://github.com/camunda/camunda/pull/27213/commits/a964fffdb3521d560f2b006248a6b1b636062669) you missed to set the `ownerType`, can you double check ?

Reply:
Not in all the classes, but I think where the authorization is enabled

Reply:
Agh! Sorry for that, too many changes. It's fixed now.